utils/utils_test.go

```diff
@@ -513,3 +514,70 @@ func TestRange(t *testing.T) {
 		})
 	}
 }
+
+func TestLineCount(t *testing.T) {
+	type args struct {
+		filePath string
+	}
+	tests := []struct {
+		name string
+		args args
+		want int
+	}{
+		{
+			name: "negative test",
+			args: args{
+				filePath: "hello",
+			},
+			want: 0,
+		}, {
+			name: "positive test",
+			args: args{
+				filePath: "./utils_test.go",
+			},
+			want: 1,
```

Comment:
Can we really read this file during `go test ...`?

Reply:
It worked while I was running the unit tests. I chose this file, because it's always here. Meaning there is no path diffrence or stuff. Do you have a better idea?

Reply:
I am curious that `want: 1` represents only 1 line for this file?

Reply:
Want atleast 1 or more. Not zero